build-antivirus-from-source.tf

```diff
@@ -5,20 +5,24 @@ resource "null_resource" "build-antivirus-from-source" {
     : 0
   )
 
+  triggers = {
+    git_revision = "${var.git-revision}"
+  }
+
   provisioner "local-exec" {
-    command = "bash ${path.module}/scripts/build-antivirus-from-source.sh"
+    command = "REVISION=${var.git-revision} bash ${path.module}/scripts/build-antivirus-from-source.sh"
   }
 }
 
 resource "aws_s3_bucket_object" "antivirus-code" {
   depends_on = [null_resource.build-antivirus-from-source]
 
   bucket = aws_s3_bucket.antivirus-code.bucket
-  key    = "lambda.zip"
+  key    = "lambda-${var.git-revision}.zip"
```

Comment:
It might be better if I had used etag rather than key to force an update to this resource, but it was annoying to test because as I understand it there is only a single working revision of the lambda at the moment.